src/css/fluid-covid-map-viz.css

```diff
@@ -0,0 +1,344 @@
+body {
+    font-family: sans-serif;
+    margin: 10px;
+}
+
+.fld-mapviz-map-holder {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+}
+
+.flc-query-holder > * {
+    margin-bottom: 7px;
+    margin-top: 4px;
+}
+
+.fld-mapviz-query {
+    background-image: url(../img/Search.svg);
+    background-position: 5px;
+    background-repeat: no-repeat;
+    border: 1px solid black;
+    border-radius: 1.5rem;
+    height: 2rem;
+    padding-left: 40px;
+    width: 700px;
+}
+
+.fld-mapviz-map {
+    height: 575px;
+    width: 800px;
+}
+
+.fld-mapviz-map-tooltip {
+    background-color: white;
+    border: 1px solid black;
+    font-size: 0.9rem;
+    position: absolute;
+    right: 20px;
+    top: 20px;
+    width: 20rem;
+    z-index: 800;
+}
+
+.fld-mapviz-map-tooltip tr:nth-child(even) {
+    background: #eee;
+}
+
+.fld-mapviz-map-tooltip tr:nth-child(odd) {
+    background: #fff;
+}
+
+.fld-mapviz-map-tooltip table {
+    border-collapse: collapse;
+}
+
+.fl-mapviz-queryheader {
```

Comment:
Css selectors from line 57-76 as well as `.fl-mapviz-filter-icon-entrances` at line 99 are not found in the markup. Can you double check if they are still in use?

Reply:
Thanks for the catch, now removed